### PR TITLE
Adds missing description, synopsis and examples for handwritten authentication cmdlets

### LINKED ISSUE #2332: Review synopsis, description, and examples for handwritten cmdlets
Some of the help files added as part of https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/2307 have placeholder synopsis, description, and examples for handwritten cmdlets. The feature shouldn't ship with placeholder values. The descriptions can be found in the C# cmdlet code and their associated PRs - https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/69f727bfff4e6d9fda7a7300d9954b1c9f3a8d5a/src/Authentication/Authentication/Cmdlets/Environment/AddMgEnvironment.cs#L17 and https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/425

cc\ @timayabi2020

### PR DESCRIPTION
**Pull request by timayabi2020**
Fixes #2332

### CODE REVIEW

**Review by calebkiage — DISMISSED** [review]

File: src/Authentication/descriptions/Set-MgRequestContext.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Sets request context for microsoft graph invocations
```
Comment (peombwa) [inline src/Authentication/descriptions/Set-MgRequestContext.md:1]: nit: Fixes company and product name casing (marketing).
```suggestion
Sets request context for Microsoft Graph invocations
```

File: src/Authentication/descriptions/Get-MgGraphOption.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Confirm if web account manager (WAM) support has been enabled.
```
Comment (peombwa) [inline src/Authentication/descriptions/Get-MgGraphOption.md:1]:
```suggestion
Gets global configurations that apply to the SDK. For example, check if Web Account Manager (WAM) support has been enabled.
```

File: src/Authentication/descriptions/Set-MgGraphOption.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Enables web account manager (WAM) support
```
Comment (peombwa) [inline src/Authentication/descriptions/Set-MgGraphOption.md:1]:
```suggestion
Sets global configurations that apply to the SDK. For example, toggle Web Account Manager (WAM) support.
```

File: src/Authentication/synopsis/Get-MgGraphOption.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Confirm if web account manager (WAM) support has been enabled.
```
Comment (peombwa) [inline src/Authentication/synopsis/Get-MgGraphOption.md:1]:
```suggestion
Gets global configurations that apply to the SDK. For example, check if Web Account Manager (WAM) support has been enabled.
```

**Review by peombwa — CHANGES_REQUESTED** [review]
Minor updates the descriptions.

**Pull request merged**